src/Artsy/Analytics/trackPageViewWrapper.tsx

```diff
@@ -11,6 +11,9 @@ export function trackPageViewWrapper<T>(
     componentDidMount() {
       trackPageView()
     }
+    componentDidUpdate() {
+      trackPageView()
+    }
```

Comment:
@mzikherman - this could greatly inflate the analytics event log if not handled correctly per react best practices, by ensuring that there are no unnecessary re-renders. Things like [React.memo](https://reactjs.org/docs/react-api.html#reactmemo) and [React.PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent) will probably need to be implemented on pages that use this HOC. 

@zephraph - any thoughts?

Reply:
In any case, it will be good to run React's [DevTools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)  over this component tree to instrument, while also checking the network logs to see if we're spamming segment.

Reply:
I did some light navigation and examining of the network log, and everything generally seemed fine. As I was reloading pages, it was firing once, and as I was client-side navigating around, it was also firing once (each and every time, which is good!).

I did check filter interactions (which didn't re-trigger anything).